xbmc/utils/RssReader.cpp

```diff
@@ -321,7 +321,7 @@ bool CRssReader::Parse(int iFeed)
     return false;
 
   std::string strValue = rootXmlNode->Value();
-  if (strValue.find("rss") == std::string::npos || strValue.find("rdf") == std::string::npos)
+  if (!(strValue.find("rss") == std::string::npos || strValue.find("rdf") == std::string::npos))
```

Comment:
The code used to be the else case of:
```
  if (strValue.find("rss") != std::string::npos ||
      strValue.find("rdf") != std::string::npos)
```
which if you want to invert (as the else case if being dropped) is:
```
  if (strValue.find("rss") == std::string::npos && strValue.find("rdf") == std::string::npos)
```
so I'm not sure this change is correct.

Reply:
Right, no idea how I could fuck that one up :tired_face:

Reply:
De Morgan shall forgive you some day :wink: